prometheus-to-sd/main.go

```diff
@@ -70,7 +72,7 @@ func main() {
 	defer glog.Flush()
 	flag.Parse()
 
-	sourceConfigs := config.SourceConfigsFromFlags(source, component, host, port, whitelisted)
+	sourceConfigs := config.SourceConfigsFromFlags(source, component, host, port, whitelisted, gaugeToCumulativeWhitelist)
```

Comment:
This thing is not going to compile and you don't need it.

Reply:
Sorry, somehow that stuck around when I was cleaning things up